Dockerfile

```diff
@@ -0,0 +1,39 @@
+# TODO: select a base image
+# Tip: start with a full base image, and then see if you can optimize with
+#      a slim or tensorflow base
+
+#      Standard version
+FROM python:3.10.10-buster
```

Comment:
Will it be a problem that we are running Python version 3.10.6 in our venv?

Reply:
It shouldn't cause any issues as 3.10.10 is used as a based image for our application container's Python version. Also minor version differences do not introduce significant changes that impact the compatibility.

Reply:
Cool, was not sure about it

Reply:
that was a good ask.